core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -932,15 +958,17 @@ private Void visitScanFilterAndProjectInfo(
                 }
             }
 
+            Collection<DynamicFilters.Descriptor> dynamicFilters = ImmutableList.of();
             if (filterNode.isPresent()) {
                 operatorName += "Filter";
                 formatString += "filterPredicate = %s, ";
                 Expression predicate = filterNode.get().getPredicate();
                 DynamicFilters.ExtractResult extractResult = extractDynamicFilters(predicate);
                 arguments.add(unresolveFunctions(combineConjunctsWithDuplicates(extractResult.getStaticConjuncts())));
                 if (!extractResult.getDynamicConjuncts().isEmpty()) {
-                    formatString += "dynamicFilter = %s, ";
-                    arguments.add(printDynamicFilters(extractResult.getDynamicConjuncts()));
+                    formatString += "dynamicFilters = %s, ";
+                    dynamicFilters = extractResult.getDynamicConjuncts();
```

Comment:
inline with next line

Reply:
I'm collecting dynamicFilters for later call to printDynamicFilterDomainStats